Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -71,9 +71,16 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath)
 if nargin<2 || isempty(PWI_Path)
     PWI_Path = x.P.Pop_Path_PWI;
 end
-if nargin<3 || isempty(OutputPath)
-    OutputPath = x.P.Pop_Path_qCBF;
+
+if nargin<3 || isempty(pathOutputCBF)
+    pathOutputCBF = x.P.Pop_Path_qCBF;
 end
+% Define output path for ATT map based on the CBF output path
+% Replace CBF with ATT in the output path
+iStringCBF = regexpi(pathOutputCBF, 'CBF');
+iStringCBF = iStringCBF(end);
+pathOutputTT = [pathOutputCBF(1:(iStringCBF-1)) 'TT' pathOutputCBF((iStringCBF+3):end)];
```

Comment:
'TT' Is this a typo?

Reply:
No it isn't - historically, we're having it this way and decided to keep it like this and only change it when BIDS-derivatives names are introduced.

Reply:
See also my reply in the re-opened comment.